config.php

```diff
@@ -685,12 +685,9 @@ function getClock($time = false, $format = false, $adjust = true) {
 	
 	// Don't touch this unless you know what you are doing. (modifying this(key value) also requires modifications in OT files /XML/commands.xml).
 	$config['ingame_positions'] = array(
-		1 => 'Player',
-		2 => 'Tutor',
-		3 => 'Senior Tutor',
-		4 => 'Gamemaster',
-		5 => 'Community Manager',
-		6 => 'God',
+		1 => 'player',
+		2 => 'gamemaster',
+		3 => 'god',
 	);
```

Comment:
On TFS 0.3/4, positions 1-6 are used, while 1-3 are used on TFS 1.0+. 
So i think this change fits TFS 1.6 better, as it will only support 1.0 later. 
Perhaps just add a PHP comment
```php
// TFS 1.0+ should use these settings: . . .
```